Integrations/UrlScan/UrlScan.py

```diff
@@ -46,13 +48,24 @@ def http_request(method, url_suffix, json=None, wait=0, retries=0):
         verify=USE_SSL
     )
     if r.status_code != 200:
+
         if r.status_code == 429:
             if retries <= 0:
                 # Error in API call to URLScan.io [429] - Too Many Requests
                 return_error('API rate limit reached. Use the retries and wait arguments when submitting multiple URls')
             else:
                 time.sleep(wait)
                 return http_request(method, url_suffix, json, wait, retries - 1)
+        response_json = r.json()
+        # raise ValueError(f'Response: {response_json}')
+        error_description = response_json.get('description')
+        should_continue_on_blacklisted_urls = demisto.args().get('continue_on_blacklisted_urls')
+        if should_continue_on_blacklisted_urls and error_description == BLACKLISTED_URL_ERROR_MESSAGE:
+            response_json['uuid'] = -1 # TODO : check what happens with that uuid and how that could be breaking
+            requested_url = JSON.loads(json)['url']
+            demisto.results(f'The URL {requested_url} is blacklisted, thus no results will be returned for it.')
```

Comment:
maybe return some context info about this?

Reply:
We can add a new context path, I don't think we can update any of the existing ones